pkg/cmd/attestation/tufrootverify/tuf-root-verify.go

```diff
@@ -0,0 +1,83 @@
+package tufrootverify
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cli/cli/v2/pkg/cmd/attestation/auth"
+	"github.com/cli/cli/v2/pkg/cmd/attestation/logging"
+	"github.com/cli/cli/v2/pkg/cmd/attestation/verification"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/sigstore/sigstore-go/pkg/tuf"
+	"github.com/spf13/cobra"
+)
+
+func NewTUFRootVerifyCmd(f *cmdutil.Factory) *cobra.Command {
+	var mirror string
+	var root string
+	var cmd = cobra.Command{
+		Use:   "tuf-root-verify --mirror <mirror-url> --root <root.json>",
+		Args:  cobra.ExactArgs(0),
+		Short: "Verify the TUF repository from a provided TUF root",
+		Long: heredoc.Docf(`
+			Verify a TUF repository with a local TUF root.
+
+			The command requires you provide the %[1]s--mirror%[1]s flag, which should be the URL 
+			of the TUF repository mirror.
+			
+			The command also requires you provide the %[1]s--root%[1]s flag, which should be the 
+			path to the TUF root file.
+
+			GitHub relies on TUF to securely deliver the trust root for our signing authority.
+			For more information on TUF, see the official documentation: <https://theupdateframework.github.io/>.
+		`, "`"),
+		Example: heredoc.Doc(`
+			# Verify the TUF repository from a provided TUF root
+			gh attestation tuf-root-verify --mirror https://tuf-repo.github.com --root /path/to/1.root.json
+		`),
+		Run: func(cmd *cobra.Command, args []string) {
+			logger := logging.NewDefaultLogger(f.IOStreams)
+
+			if err := auth.IsHostSupported(); err != nil {
+				fmt.Sprintln(logger.IO.Out, logger.ColorScheme.Red(err.Error()))
+				os.Exit(1)
+			}
+
+			if err := verifyTUFRoot(mirror, root); err != nil {
+				fmt.Sprintln(logger.IO.Out, logger.ColorScheme.Redf("Failed to verify the TUF repository: %s", err))
+				os.Exit(1)
+			}
+			fmt.Sprintln(logger.IO.Out, logger.ColorScheme.Green("Successfully verified the TUF repository"))
+		},
+	}
+
+	cmd.Flags().StringVarP(&mirror, "mirror", "m", "", "URL to the TUF repository mirror")
+	cmd.MarkFlagRequired("mirror") //nolint:errcheck
+	cmd.Flags().StringVarP(&root, "root", "r", "", "Path to the TUF root file on disk")
+	cmd.MarkFlagRequired("root") //nolint:errcheck
+
+	return &cmd
+}
+
+func verifyTUFRoot(mirror, root string) error {
```

Comment:
Nit: The command and everything else above is the flipped version of this; `tuf-root-verify` vs `verify-tuf-root`.  My personal philosophy is that inconsistency will trip up those who maintain the code.

Reply:
Whoops, that's an oversight. I'll update the func name.